packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -104,6 +105,7 @@ class GoogleMap extends StatefulWidget {
     this.zoomGesturesEnabled = true,
     this.liteModeEnabled = false,
     this.tiltGesturesEnabled = true,
+    this.fortyFiveDegreeImageryEnabled = false,
```

Comment:
If we're going to add multiple web-only settings, perhaps we should have a new `WebSettings` class to encapsulate them, so this method doesn't continue growing without bounds. (In general, it seems like we'd benefit from better organization of params here, so we may as well start now.)

The main reason not to do that would be if we believe this would ever be a non-web option. Is there any indication about that in the APIs?

Reply:
Here is our original discussion on this issue: https://github.com/flutter/plugins/pull/4916#discussion_r879941100

I'm not sure we ever came to the conclusion that this option would or would not exist on other platforms